### PR TITLE
Tasks jesse - Added some comments in and implemented edit task functionality

### PR DESCRIPTION
**Pull request by jessesmith-13**

### CODE REVIEW

**Review by johnkwak08 — COMMENTED** [review]
love the comments!

File: client/components/Tasks/EditTasks.jsx [hunk 1]
```diff
@@ -63,38 +61,28 @@ const AddTasks = ({rerenderCalendar, rerenderData}) => {
     }
     console.log(newTask);
```
Comment (mingwencode) [inline client/components/Tasks/EditTasks.jsx:62]: console.log

Comment (greta-schock) [inline client/components/Tasks/EditTasks.jsx:62]: rogue console.log

File: client/components/Tasks/EditTasks.jsx [hunk 2]
```diff
@@ -63,38 +61,28 @@ const AddTasks = ({rerenderCalendar, rerenderData}) => {
...
+    const currentSmove = userData.smoves.filter(smove => smove.isCurrentSmove)[0];
+    for (let i = 0; i < currentSmove.tasks.length; i++) {
+      if (currentSmove.tasks[i].taskName === oldName) {
+        currentSmove.tasks[i] = newTask;
+      }
     }
 
     axios.patch(`/user/${userData.email}`, {data: userData.smoves})
       .then(({ data }) => {
         console.log('new task added: ', data)
```
Comment (mingwencode) [inline client/components/Tasks/EditTasks.jsx:73]: console.log

Comment (greta-schock) [inline client/components/Tasks/EditTasks.jsx:73]: rogue console.log

**Review by greta-schock — CHANGES_REQUESTED** [review]
Console.logs

**Pull request merged**